digitalpy/core/component_management/controllers/component_management_controller.py

```diff
@@ -7,21 +7,22 @@
 # Original author: Giu Platania
 # 
 #######################################################
-from Catalog.Implementation.Libraries.Digitalpy.digitalpy.Async.logic.impl.default_business_rule_controller import DefaultBusinessRuleController
+from digitalpy.core.logic.impl.default_business_rule_controller import DefaultBusinessRuleController
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position